modules/updateTask.js

```diff
@@ -0,0 +1,15 @@
+import { getTaskFromLocalStorage, setTaskIntoLocalStorage } from './localStorage.js';
+
+const updateTask = (e) => {
+  const tasks = getTaskFromLocalStorage();
+  const selectedID = parseInt(e.id, 10);
+
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].index === selectedID) {
+      tasks[i].description = e.value;
+      break;
+    }
+  }
+  setTaskIntoLocalStorage(tasks);
+};
+export default updateTask;
```

Comment:
As listed in the project requirements `A function for editing task descriptions is implemented.`

- [ ] Update or edit is not working as expected.

![image](https://user-images.githubusercontent.com/55635977/159741218-be34a17f-e42c-440c-8f29-adc280d20359.png)

Reply:
To the next reviewer, after a call with a student, I found that the issue was my chrome browser so I had to delete the caches and the project is working perfectly.